src/setup.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const { User } = require('./models/User');
+const { Expense } = require('./models/Expense');
+
+User.sync({ force: true });
```

Comment:
Consider using `alter: true` instead of `force: true` to avoid data loss when syncing models. 🚀